### ISSUE TITLE
Get WCSAxes included in astropy 1.0

### ISSUE THREAD

**Issue by Cadair** [astrofrog/wcsaxes#140]
This is a bit of a general issue, but from my perspective I would love to see this in 1.0. Obviously there are some decisions to be made on the astropy side, but this issue is an attempt to formulate a list of things that need to be done before a merge into astropy would be possible.

**Comment by hamogu** [comment]
I've been using WCSAxes for a few things now and it seems to work really well.
It's too late for astropy 1.0 by now, but I want to advocate to integrate it into astropy sooner rather than later.
(Yes, there are a few issues open, but to me non of them seem like "absolutely must have".)

**Comment by astrofrog** [comment]
I've now made it so the test baseline images are no longer in the repository but on data.astropy.org, which is a first step towards this. I've then sent a message to astropy-dev about how we might want to approach this and adding other parts of functionality in future:

https://groups.google.com/forum/#!topic/astropy-dev/83YqYAWeCis

**Comment by astrofrog** [comment]
Yay this was done!
